CHANGELOG.md

```diff
@@ -4,19 +4,22 @@
 
 ### Recommended changes
 
-#### Remove the tabindex attribute from the Error Summary component
+We've recently made some changes to GOV.UK Frontend. While these are not breaking changes, implementing them will make your service work better.
 
-If you're not using Nunjucks macros, remove the `tabindex` attribute from the HTML for the Error Summary component. This attribute is now added and removed by the JavaScript.
+#### Remove the `tabindex` attribute from the error summary component
 
-This change was introduced in [#2491: Prevent error summary from being re-focused after it has been initially focused on page load](https://github.com/alphagov/govuk-frontend/pull/2491).
+If you're not using Nunjucks macros, remove the `tabindex` attribute from the error summary's HTML. The component JavaScript now adds and removes this attribute.
+
+This change was introduced in [pull request #2491: Prevent error summary from being re-focused after it has been initially focused on page load](https://github.com/alphagov/govuk-frontend/pull/2491).
 
 ### Fixes
 
-- [#2475: Tweak whitespace in input component HTML for improved readability](https://github.com/alphagov/govuk-frontend/pull/2475)
-- [#2491: Prevent error summary from being re-focused after it has been initially focused on page load](https://github.com/alphagov/govuk-frontend/pull/2491)
+We’ve made fixes to GOV.UK Frontend in the following pull requests:
+
+- [#2475: Tweak whitespace in input component HTML to improve readability](https://github.com/alphagov/govuk-frontend/pull/2475)
```

Comment:
If this change is ok, I'll also change the title of the linked PR.

Reply:
Should this be 'text input' instead of 'input'?

Maybe we could change the PR title to something like 'Tweak whitespace in HTML for text input component'.

Reply:
We use "text input" in the Design System, so probably best to be consistent :+1: